website/content/docs/commands/token/create.mdx

```diff
@@ -11,23 +11,123 @@ description: |-
 
 # token create
 
-The `token create` command creates a new token that can be used for
-authentication. This token will be created as a child of the currently
-authenticated token. The generated token will inherit all policies and
-permissions of the currently authenticated token unless you explicitly define a
-subset list policies to assign to the token.
+Creates a new token that you can use to authenticate to Vault.
 
-A ttl can also be associated with the token. If a ttl is not associated with the
-token, then it cannot be renewed. If a ttl is associated with the token, it will
-expire after that amount of time unless it is renewed.
+<CodeBlockConfig hideClipboard>
 
-Metadata associated with the token (specified with `-metadata`) is written to
-the audit log when the token is used.
+```shell-session
+$ vault token create [flags]
+
+$ vault token create [-help | -h]
+```
+
+</CodeBlockConfig>
+
+## Description
+
+`token create` creates a new token that you can use to authenticate to Vault. 
+Vault will create this token as a child of the currently authenticated token. 
+The generated token will inherit all policies and permissions of the currently 
+authenticated token unless you explicitly define a subset list policies to 
+assign to the token.
+
+You can also associate a TTL with the token. If you do specify a TTL, the token 
+will expire after that amount of time unless it is renewed. Otherwise, if you 
+do not specify a TTL, the token cannot be renewed.
+
+Vault writes metadata associated with the token (specified with `-metadata`) 
+to the audit log when the token is used.
 
 If a role is specified, the role may override parameters specified here.
 
+## Command arguments
+
+None.
+
+## Command options
+
+None.
+
+## Command flags
+
+@include 'cli/token/flags/display-name.mdx'
+
+@include 'cli/token/flags/entity-alias.mdx'
+
+<br /><hr /><br />
+
+@include 'cli/token/flags/explicit-max-ttl.mdx' 
+
+<br /><hr /><br />
+
+@include 'cli/token/flags/id.mdx' 
+
+<br /><hr /><br />
+
+@include 'cli/token/flags/metadata.mdx'
+
+<br /><hr /><br />
+
+@include 'cli/token/flags/no-default-policy.mdx'
+
+<br /><hr /><br />
+
+@include 'cli/token/flags/orphan.mdx'
+
+<br /><hr /><br />
+
+@include 'cli/token/flags/period.mdx'
+
+<br /><hr /><br />
+
+@include 'cli/token/flags/policy.mdx'
+
+<br /><hr /><br />
+
+@include 'cli/token/flags/renewable.mdx'
+
+<br /><hr /><br />
+
+@include 'cli/token/flags/role.mdx'
+
+<br /><hr /><br />
+
+@include 'cli/token/flags/ttl.mdx'
+
+<br /><hr /><br />
+
+@include 'cli/token/flags/type.mdx'
+
+<br /><hr /><br />
+
+@include 'cli/token/flags/use-limit.mdx'
+
+<br /><hr /><br />
+
+@include 'cli/token/flags/wrap-ttl.mdx'
+
+## Standard flags
+
+<br />
+
+@include 'cli/shared/flags/field.mdx'
```

Comment:
Shared flags aren't global. They're still considered command-specific flags, they are just shared across more than one command family.

Reply:
should we move this to another section since these are output flags rather than command flags?